lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -61,6 +61,12 @@ def execute(command, options = {})
         command_runner.output
       end
 
+      def response_on_execute(command, options = {})
```

Comment:
I would not know from the name what to expect from the name. What about `execute_with_status`?